packages/core/src/validation.ts

```diff
@@ -176,6 +176,7 @@ const accountCenter = Joi.object({
   position: commonPositions,
   expanded: Joi.boolean(),
   minimal: Joi.boolean(),
+  disableProtectedRpc: Joi.boolean(),
```

Comment:
I think you need to add disableProtectedRpc to accountCenterInitOptions too- I couldn't get it to initialize with disabledProtectedRpc: true otherwise

Reply:
Good catch! Forgot to commit those changes!